src/bootstrap/src/core/build_steps/compile.rs

```diff
@@ -507,8 +514,8 @@ pub fn std_cargo(builder: &Builder<'_>, target: TargetSelection, stage: u32, car
         );
         let compiler_builtins_root = builder.src.join("src/llvm-project/compiler-rt");
         assert!(compiler_builtins_root.exists());
-        // Note that `libprofiler_builtins/build.rs` also computes this so if
-        // you're changing something here please also change that.
+        // The path to `compiler-rt` is also used by `profiler_builtins` (above),
+        // so if you're changing something here please also change that as appropriate.
         cargo.env("RUST_COMPILER_RT_ROOT", &compiler_builtins_root);
```

Comment:
We could probably use the same mechanism also for `optimized_compiler_builtins`. But that flag is probably much more niche than using the profiler, so let's just keep it as is for now.

Reply:
Yeah, I kept them separate for now to avoid scope creep and to avoid any risk of breaking the compiler builtins.

But if this approach works well (and I hope it does) then extending it to optimized compiler builtins in the future sounds reasonable.